e2e/tests/components/page-header/page-header.e2e-spec.ts

```diff
@@ -268,4 +269,24 @@ describe('Page Header Tests', () => {
         expect(await page.getSubheaderText(page.pageHeader1)).toBe('My Subheader');
 
     });
+
+    it('should add a href to breadcrumbs that specify a routerLink', async () => {
+        expect(await page.getBreadcrumbHref(0)).toContain('#/page-header/home');
+    });
+
+    it('should not add a href to breadcrumbs that do not specify a routerLink', async () => {
+        expect(await page.getBreadcrumbHref(1)).toBeNull();
+    });

```

Comment:
These two tests are duplicates of the karma tests, so we could leave them out. The goal of e2e tests is to assert the end user's experience of a feature, e.g. "I click on x and y happens."